.infra/crons.ts

```diff
@@ -100,4 +100,8 @@ export const crons: Cron[] = [
     name: 'update-current-streak',
     schedule: '30 * * * *',
   },
+  {
+    name: 'sync-subscription',
+    schedule: '*/5 * * * *',
```

Comment:
Maybe something more specific to what it does, subscriptions is a bit broad term. Also does it manage to execute in 5min, we don't want them to overlap?

Reply:
I renamed it. And I think 5 minutes is enough, can adjust it once in production and see how long an execution takes.

Reply:
👌